src/common/config_opts.h

```diff
@@ -874,6 +874,9 @@ OPTION(osd_snap_trim_priority, OPT_U32, 5)
 OPTION(osd_snap_trim_cost, OPT_U32, 1<<20) // set default cost equal to 1MB io
 
 OPTION(osd_scrub_priority, OPT_U32, 5)
+// set default must scrub priority higher than scrub priority to make the
+// requested scrubs jump the queue of scheduled scrubs
+OPTION(osd_must_scrub_priority, OPT_U32, 6)
```

Comment:
@athanatos thank you for your comment, updated to osd_requested_scrub_priority now. ;)